command_line/commoncommandlinefunctions.cpp

```diff
@@ -1048,10 +1046,6 @@ NodeColourScheme getColourSchemeOption(QString option, QStringList * arguments)
         return UNIFORM_COLOURS;
     else if (colourString == "depth")
         return DEPTH_COLOUR;
-    else if (colourString == "blastsolid")
```

Comment:
How we'd support annotations via cmdline now?

Reply:
For now I saved backward compatibility for "--colour blastsolid", "--colour blastrainbow" and "--blasthits" in CLI. In future we need to implement a common way to set annotation setting from CLI.